nupic/frameworks/opf/metrics.py

```diff
@@ -1459,7 +1424,63 @@ def addInstance(self, groundTruth, prediction, record = None):
 
     return self.aggregateError
 
+###################################################################################
+class MetricMulti(MetricsIface):
+  """Multi metric can combine multiple other (sub)metrics and 
+     weight them to provide combined score."""
 
+  def __init__(self, metricSpec):
+    """MetricMulti constructor using metricSpec is not allowed."""
+    raise ValueError("MetricMulti cannot be constructed from metricSpec string! "
+                     "Use MetricMulti(id,weights,metrics) constructor instead.")
```

Comment:
The order of arguments here isn't consistent with the order in the actual constructor below.